stripe/src/test/java/com/stripe/android/view/CardInputWidgetTest.java

```diff
@@ -339,19 +339,24 @@ public void onUpdateText_forAmExPrefix_callsSetImageResourceAndSetsLengthOnCvc()
     @Test
     public void updateToInitialSizes_returnsExpectedValues() {
         // Initial spacing should look like
-        // |---total == 500--------|
-        // |(card==190)--(space==260)--(date=50)|
+        // |img==60||---total == 500--------|
```

Comment:
How exactly do you arrive at these test numbers? Aren't they dependent on the resolution of the screen which they get rendered on? (Does the test suite assume some default screen resolution or something like that?)